tests/testthat/test-bundlePackage.R

```diff
@@ -1,18 +1,41 @@
 test_that("can snapshot deps with renv", {
-  app_dir <- local_temp_app(list("foo.R" = "library(MASS)"))
+  app_dir <- local_temp_app(list(foo.R = "library(foreign); library(MASS)"))
+
   expect_snapshot(pkgs <- bundlePackages(app_dir))
-  expect_named(pkgs, "MASS")
+
+  # renv is not a dependency
+  expect_named(pkgs, c("foreign", "MASS"), ignore.order = TRUE)
+  expect_named(pkgs$foreign, c("Source", "Repository", "description"))
   expect_named(pkgs$MASS, c("Source", "Repository", "description"))
 
   # No renv lockfile left behind
   expect_equal(list.files(app_dir), "foo.R")
 })
 
-test_that("can snapshot deps with packrat", {
+test_that("can snapshot deps with packrat (option)", {
```

Comment:
IMO you don't need this test because you test `usePackrat()` below

Reply:
I'm going to keep both variations because `usePackrat` is an implementation detail and the setting/env var are incoming state.. and this is a reasonable point in the stack to run these functional tests. The `usePackrat` tests are used to check other state permutations; we won't re-check all those situations with renv/packrat analysis.